pkg/network/usm/kafka_monitor_test.go

```diff
@@ -168,6 +190,9 @@ func (s *KafkaProtocolParsingSuite) testKafkaProtocolParsing(t *testing.T, tls b
 		return getDefaultTestConfiguration(tls)
 	}
 
+	tmp, _ := version.LookupMaxKeyVersion(kafka.FetchAPIKey)
```

Comment:
Could you incorporate handling for the scenario where the function returns false?